### PR TITLE
chore(css): Move CSS examples - Shapes, transforms, transition, display

### PR DESCRIPTION
**Pull request by bsmth**
### Description

As part of an initiative to reduce repo maintenance & code duplication, this PR moves some external examples into content. This PR converts the following macros to live samples:

```
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/inset.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/inset-box.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/circle.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/circle-generated.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/ellipse.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/ellipse-keywords.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/basic-shape/polygon.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/box/margin-box.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/box/border-box.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/box/padding-box.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/box/content-box.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/box/bottom-margin-box.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/circle.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/box.html", '100%', 810)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/image.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/threshold.html", '100%', 820)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/shape-margin.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/generated-content.html", '100%', 850)}}
{{EmbedGHLiveSample("css-examples/shapes/overview/clip-path.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/image/simple-example.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/image/margin.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/image/threshold.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/image/generated-content.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/image/gradient.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/shapes/image/radial-gradient.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/modules/transforms.html", '100%', 900)}}
{{EmbedGHLiveSample("css-examples/transitions/js-transitions.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/display/multi-keyword/span-flex.html", '100%', 440)}}
{{EmbedGHLiveSample("css-examples/display/multi-keyword/inline-flex.html", '100%', 440)}}
{{EmbedGHLiveSample("css-examples/display/multi-keyword/multi-keyword-flex.html", '100%', 640)}}
{{EmbedGHLiveSample("css-examples/display/multi-keyword/inline-block.html", '100%', 440)}}
```

### Motivation

Motivation is described in the project issue.

### Related issues and pull requests

- https://github.com/mdn/mdn/issues/597

## TODO

- [x] Add star to shared assets repo https://github.com/mdn/shared-assets/pull/32
- [x] Replace star using shared version

### CODE REVIEW

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hey @bsmth! I've looked through all of these. I can verify that they (at least mostly) display correctly by running them in MDN Playground. However, they are not displaying on the preview pages, which is a bit strange.

File: files/en-us/web/css/css_shapes/basic_shapes/index.md [hunk 1]
```diff
@@ -80,11 +80,81 @@ The above rules can therefore also be written as:
...
+  shape-outside: inset(20px 50px 10px 0 round 50px);
+  background-color: rebeccapurple;
+  border: 2px solid black;
+  border-radius: 10px;
+  margin: 20px;
+  padding: 20px;
+}
+```
+
+{{EmbedLiveSample("inset", "", "250px")}}
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/css/css_shapes/basic_shapes/index.md:118]: That is strange, I did notice the output box was empty occasionally on localhost, but had content on refresh, I wonder if it's a Playground problem with lazy loading / order of loading resources. Needs some investigation.

[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/css/css_shapes/basic_shapes/index.md:118]: @chrisdavidmills this is a known issue due to changes in Playground envs, it should be resolved in the coming days.

[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/css/css_shapes/basic_shapes/index.md:118]: OK. In that case, I'm happy with all the additions, except where commented.

File: files/en-us/web/css/css_shapes/shapes_from_images/index.md [hunk 2]
```diff
@@ -14,11 +14,72 @@ To use an image for creating a shape, the image needs to have an Alpha Channel,
...
+
+```css live-sample___margin
+body {
+  font: 1.2em / 1.5 sans-serif;
+}
+
+img {
+  float: left;
+  shape-outside: url(https://mdn.github.io/shared-assets/images/examples/star-shape.png);
+  shape-image-threshold: 0.2;
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/css/css_shapes/shapes_from_images/index.md]: Correct! This is wrong in the original, fixed in c22e6d2e333a4eccaf489e777626a21587d22d3f

[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/css/css_shapes/shapes_from_images/index.md]: Cool, that looks fixed now in the code listing in the page.

Although...now the live example doesn't appear to render in the page, or in Playground... ;-§

File: files/en-us/web/css/css_shapes/shapes_from_images/index.md [hunk 3]
```diff
@@ -34,15 +95,77 @@ The {{cssxref("shape-image-threshold")}} property enables the creation of shapes
...
+```
+
+```css live-sample___threshold
+body {
+  font: 1.2em / 1.5 sans-serif;
+}
+
+img {
+  float: left;
+  shape-outside: url(https://mdn.github.io/shared-assets/images/examples/star-shape.png);
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/css/css_shapes/shapes_from_images/index.md]: Again, looks correct on the page code listing, but I can't see it rendered in the live example in the page, or in Playground.

**Review by chrisdavidmills — APPROVED** [review]
@bsmth looks fixed, so approving, although I can't see the examples rendered anywhere.

**Comment by bsmth** [comment]
> @bsmth looks fixed, so approving, although I can't see the examples rendered anywhere.

Thanks a lot! The playground in preview builds is probably not going to be fixed until next week given Rari release prio, so we can ignore for now as Playground is working on prod (& localhost). I'll double check on the branch, but we should be good to go!

**Pull request merged**